vspace/src/main/java/edu/asu/diging/vspace/core/model/ModuleOverview.java

```diff
@@ -0,0 +1,30 @@
+package edu.asu.diging.vspace.core.model;
```

Comment:
This package contains the persistable data model. This class does not belong here.

Reply:
have changed this to the package edu.asu.diging.vspace.core.services.impl.model as i see that the classes here contain the service model implementations